domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -8,8 +8,10 @@ import no.nav.su.se.bakover.domain.oppdrag.Oppdrag
 import java.time.Clock
 import java.util.UUID
 
+data class Saksnummer(val nummer: Int)
```

Comment:
Er det noe poeng og spare på bitsa internt? Sjelden jeg bruker Int's som IDer bare fordi `vi aldri kommer til å få 2 mill entries`.

Reply:
Ingen speciell grunn till det. Tänker du att den borde vara `Long` eller så? 😊

Reply:
Just det! (såfremt den matcher basen sin type da)

Reply:
Den är int i basen atm 🙈 men max verdin på en int är ikke 2 mill men mer än 2 miljarder 🙉

Reply:
Hehe, stemmer, hvis den er int i basen, er det vel enklest  og let it be

Reply:
Har dock ändrat det nå så skal vara `bigint` i basen och `Long` i klassen. (Skal vara samma storlek på de )